package.json

```diff
@@ -18,7 +18,7 @@
     "docker_down": "docker-compose down",
     "docker_build": "docker-compose up -d --build",
     "docker_clean_db": "docker-compose down && docker volume rm educado-backend_mongodb_data && docker-compose up -d --build mongodb",
-    "linting": "eslint --ext .js,.jsx,.ts,.tsx . --fix",
+    "lint": "eslint --fix .",
```

Comment:
Should we force the --fix for everyone?

Reply:
yes

Reply:
Why?

Reply:
@Jatewo It is fixed, now there are two seperate, one with fix and one without